spec/features/hiring_staff_can_copy_a_vacancy_spec.rb

```diff
@@ -15,7 +15,7 @@
     new_vacancy.starts_on = 35.days.from_now
     new_vacancy.ends_on = 100.days.from_now
     new_vacancy.publish_on = 0.days.from_now
-    new_vacancy.expires_on = 30.days.from_now
+    new_vacancy.expiry_time = new_vacancy.expires_on = 30.days.from_now
```

Comment:
What does this do? `something` = `something_else` = `somevalue` looks new to me. 😄

Reply:
Its [multiple assignment](https://irb.rocks/multiple-assignments-in-ruby/), which is usually not the best practice, but its totally fine in this case because time is immutable:
![image](https://user-images.githubusercontent.com/22743709/64723928-591b7f00-d4c9-11e9-885c-ce5bd886e37f.png)